var/spack/repos/builtin/packages/git/package.py

```diff
@@ -305,3 +305,14 @@ def install_manpages(self):
             install_tree('man1', prefix.share.man.man1)
             install_tree('man5', prefix.share.man.man5)
             install_tree('man7', prefix.share.man.man7)
+
+    def setup_run_environment(self, env):
+        # Setup run environment if using SVN extension
+        # Libs from perl-alien-svn and apr-util are required in
+        # LD_LIBRARY_PATH
+        # TODO: extend to other platforms
+        if "+svn" in self.spec and sys.platform.startswith('linux'):
```

Comment:
```suggestion
        if "+svn platform=linux" in self.spec:
```

Reply:
Is it true that "+svn platform=linux" will be together in that string? What if a user also specified git+svn+tcltk, would that suggestion still apply?

Reply:
It's not looking for an exact match of that string, it's looking to see if the spec satisfies that constraint.

Reply:
Oh right makes sense, spec is a class.